diff --git a/index.php b/index.php
index f83bf90..bb3b202 100644
--- a/index.php
+++ b/index.php
@@ -83,7 +83,13 @@
         $pet2 = 'Spark Pug';
         $pet3 = 'Pico de Gato';
 
-        $pancake = array('Pancake the Bulldog', '1 year', 9, 'Lorem Ipsum', 'pancake.png');
+        $pancake = array(
+            'name' => 'Pancake the Bulldog',
+            'age'  => '1 year',
+            'weight' => 9,
+            'bio' => 'Lorem Ipsum',
+            'filename' => 'pancake.png'
+        );
 
         $pets = array($pet1, $pet2, $pet3);
     ?>
@@ -91,17 +97,17 @@
     <div class="container">
         <div class="row">
             <div class="col-lg-4 pet-list-item">
-                <h2><?php echo $pancake[0]; ?></h2>
+                <h2><?php echo $pancake['name']; ?></h2>
 
-                <img src="/images/<?php echo $pancake[4]; ?>" class="img-rounded">
+                <img src="/images/<?php echo $pancake['filename']; ?>" class="img-rounded">
 
                 <blockquote class="pet-details">
-                    <?php echo $pancake[1]; ?>
-                    <?php echo $pancake[2]; ?> lbs
+                    <?php echo $pancake['age']; ?>
+                    <?php echo $pancake['weight']; ?> lbs
                 </blockquote>
 
                 <p>
-                    <?php echo $pancake[3]; ?>
+                    <?php echo $pancake['bio']; ?>
                 </p>
             </div>
 
