diff --git a/index.php b/index.php
index aa1aeb8..03674b4 100644
--- a/index.php
+++ b/index.php
@@ -67,13 +67,12 @@
         <div class="container">
             <?php
                 $cleverWelcomeMessage = 'All the love, none of the crap!';
+                $pupCount = 5000;
             ?>
 
             <h1><?php echo $cleverWelcomeMessage; ?></h1>
 
-            <p>This is a template for a simple marketing or informational website. It includes a large callout called the
-                hero unit and three supporting pieces of content. Use it as a starting point to create something more
-                unique.</p>
+            <p>With over <?php echo $pupCount ?> pet friends!</p>
 
             <p><a class="btn btn-primary btn-lg">Learn more &raquo;</a></p>
         </div>
