diff --git a/pets_new.php b/pets_new.php
index 60e7275..4116513 100644
--- a/pets_new.php
+++ b/pets_new.php
@@ -5,9 +5,23 @@ if (isset($_POST['name'])) {
     $name = 'A dog without a name';
 }
 
-$breed = $_POST['breed'];
-$weight = $_POST['weight'];
-$bio = $_POST['bio'];
+if (isset($_POST['breed'])) {
+    $breed = $_POST['breed'];
+} else {
+    $breed = '';
+}
+
+if (isset($_POST['weight'])) {
+    $weight = $_POST['weight'];
+} else {
+    $weight = '';
+}
+
+if (isset($_POST['bio'])) {
+    $bio = $_POST['bio'];
+} else {
+    $bio = '';
+}
 
 var_dump($name, $breed, $weight, $bio);die;
 ?>
