diff --git a/pets_new.php b/pets_new.php
index 06ccb3c..a907c28 100644
--- a/pets_new.php
+++ b/pets_new.php
@@ -5,22 +5,26 @@
         <div class="col-xs-6">
             <h1>Add your Pet! Squirrel!</h1>
 
-        <div class="form-group">
-            <label for="pet-name" class="control-label">Pet Name</label>
-            <input type="text" name="name" id="pet-name" class="form-control" />
-        </div>
-        <div class="form-group">
-            <label for="pet-breed" class="control-label">Breed</label>
-            <input type="text" name="breed" id="pet-breed" class="form-control" />
-        </div>
-        <div class="form-group">
-            <label for="pet-weight" class="control-label">Weight (lbs)</label>
-            <input type="number" name="weight" id="pet-weight" class="form-control" />
-        </div>
-        <div class="form-group">
-            <label for="pet-bio" class="control-label">Pet Bio</label>
-            <textarea name="bio" id="pet-bio" class="form-control"></textarea>
-        </div>
+            <form action="/pets_new.php" method="POST">
+                <div class="form-group">
+                    <label for="pet-name" class="control-label">Pet Name</label>
+                    <input type="text" name="name" id="pet-name" class="form-control" />
+                </div>
+                <div class="form-group">
+                    <label for="pet-breed" class="control-label">Breed</label>
+                    <input type="text" name="breed" id="pet-breed" class="form-control" />
+                </div>
+                <div class="form-group">
+                    <label for="pet-weight" class="control-label">Weight (lbs)</label>
+                    <input type="number" name="weight" id="pet-weight" class="form-control" />
+                </div>
+                <div class="form-group">
+                    <label for="pet-bio" class="control-label">Pet Bio</label>
+                    <textarea name="bio" id="pet-bio" class="form-control"></textarea>
+                </div>
+
+                <button type="submit" class="btn btn-primary"><span class="glyphicon glyphicon-heart"></span> Add</button>
+            </form>
 
         </div>
     </div>
