diff --git a/pets_new.php b/pets_new.php
index c982c6b..13af54b 100644
--- a/pets_new.php
+++ b/pets_new.php
@@ -1,31 +1,33 @@
 <?php
-var_dump($_SERVER);die;
 
-if (isset($_POST['name'])) {
-    $name = $_POST['name'];
-} else {
-    $name = 'A dog without a name';
-}
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
 
-if (isset($_POST['breed'])) {
-    $breed = $_POST['breed'];
-} else {
-    $breed = '';
-}
+    if (isset($_POST['name'])) {
+        $name = $_POST['name'];
+    } else {
+        $name = 'A dog without a name';
+    }
 
-if (isset($_POST['weight'])) {
-    $weight = $_POST['weight'];
-} else {
-    $weight = '';
-}
+    if (isset($_POST['breed'])) {
+        $breed = $_POST['breed'];
+    } else {
+        $breed = '';
+    }
 
-if (isset($_POST['bio'])) {
-    $bio = $_POST['bio'];
-} else {
-    $bio = '';
-}
+    if (isset($_POST['weight'])) {
+        $weight = $_POST['weight'];
+    } else {
+        $weight = '';
+    }
+
+    if (isset($_POST['bio'])) {
+        $bio = $_POST['bio'];
+    } else {
+        $bio = '';
+    }
 
-var_dump($name, $breed, $weight, $bio);die;
+    var_dump($name, $breed, $weight, $bio);die;
+}
 ?>
 
 <?php require 'layout/header.php'; ?>
