diff --git a/layout/header.php b/layout/header.php
index cbaa157..24cc388 100644
--- a/layout/header.php
+++ b/layout/header.php
@@ -33,8 +33,8 @@
             </div>
             <div class="navbar-collapse collapse">
                 <ul class="nav navbar-nav">
-                    <li class="active"><a href="#">Home</a></li>
-                    <li><a href="#about">About</a></li>
+                    <li class="active"><a href="/">Home</a></li>
+                    <li><a href="/pets_new.php">Post</a></li>
                     <li><a href="#contact">Contact</a></li>
                     <li class="dropdown">
                         <a href="#" class="dropdown-toggle" data-toggle="dropdown">Dropdown <b class="caret"></b></a>
