diff --git a/index.php b/index.php
index a3de59b..e5b32f5 100644
--- a/index.php
+++ b/index.php
@@ -70,7 +70,7 @@
                 $pupCount = rand(50, 100);
             ?>
 
-            <h1><?php echo strrev(strtolower($cleverWelcomeMessage)); ?></h1>
+            <h1><?php echo strtoupper(strtolower($cleverWelcomeMessage)); ?></h1>
 
             <p>With over <?php echo $pupCount ?> pet friends!</p>
 
