diff --git a/index.php b/index.php
index 6fa6b89..f2f6956 100644
--- a/index.php
+++ b/index.php
@@ -96,6 +96,8 @@
                         if (array_key_exists('age', $cutePet)) {
                             if ($cutePet['age'] != '') {
                                 echo $cutePet['age'];
+                            } else {
+                                echo 'Unknown';
                             }
                         } else {
                             echo 'Unknown';
