diff --git a/data/pets.json b/data/pets.json
index e0af0bf..81caaca 100644
--- a/data/pets.json
+++ b/data/pets.json
@@ -10,7 +10,7 @@
     {
         "name": "Spark Pug",
         "breed": "Pug",
-        "age": "1.5 years",
+        "age": "hidden",
         "weight": 11,
         "bio": "You want to go to the dog park in style? Then I am your pug!",
         "image": "pet2.png"
diff --git a/index.php b/index.php
index 3014524..b623333 100644
--- a/index.php
+++ b/index.php
@@ -94,7 +94,11 @@
                         <span class="label label-info"><?php echo $cutePet['breed']; ?></span>
                         <?php
                         if (array_key_exists('age', $cutePet) && $cutePet['age'] != '') {
-                            echo $cutePet['age'];
+                            if ($cutePet['age'] == 'hidden') {
+                                echo '(contact owner for age)';
+                            } else {
+                                echo $cutePet['age'];
+                            }
                         } else {
                             echo 'Unknown';
                         }
