diff --git a/index.php b/index.php
index f2842a1..22afafe 100644
--- a/index.php
+++ b/index.php
@@ -89,10 +89,10 @@
     <div class="container">
         <div class="row">
             <?php
-                foreach ($pets as $pet) {
+                foreach ($pets as $cutePet) {
                     echo '<div class="col-lg-4">';
                     echo '<h2>';
-                    echo $pet;
+                    echo $cutePet;
                     echo '</h2>';
                     echo '</div>';
                 }
