diff --git a/index.php b/index.php
index 639ab62..b4fae86 100644
--- a/index.php
+++ b/index.php
@@ -78,10 +78,16 @@
         </div>
     </div>
 
+    <?php
+        $pet1 = 'Chew Barka';
+        $pet2 = 'Spark Pug';
+        $pet3 = 'Pico de Gato';
+    ?>
+
     <div class="container">
         <div class="row">
             <div class="col-lg-4">
-                <h2>Heading</h2>
+                <h2><?php echo $pet1; ?></h2>
 
                 <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris
                     condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis
@@ -90,7 +96,7 @@
                 <p><a class="btn btn-default" href="#">View details &raquo;</a></p>
             </div>
             <div class="col-lg-4">
-                <h2>Heading</h2>
+                <h2><?php echo $pet2; ?></h2>
 
                 <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris
                     condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis
@@ -99,7 +105,7 @@
                 <p><a class="btn btn-default" href="#">View details &raquo;</a></p>
             </div>
             <div class="col-lg-4">
-                <h2>Heading</h2>
+                <h2><?php echo $pet3; ?></h2>
 
                 <p>Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Vestibulum id ligula
                     porta felis euismod semper. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut
