diff --git a/contact.php b/contact.php
index 4836a84..804e77a 100644
--- a/contact.php
+++ b/contact.php
@@ -4,7 +4,7 @@ require 'lib/functions.php';
 <?php require 'layout/header.php'; ?>
 
 <h1>
-    Helping you find your new best friend from over <?php echo count(get_pets()); ?> pets.
+    Helping you find your new best friend from over <?php echo count(get_pets(0)); ?> pets.
 </h1>
 
 <?php require 'layout/footer.php'; ?>
diff --git a/lib/functions.php b/lib/functions.php
index e403f9c..9a6ca4b 100644
--- a/lib/functions.php
+++ b/lib/functions.php
@@ -10,7 +10,11 @@ function get_pets($limit)
         $config['database_pass']
     );
     // THIS IS A HUGE SECURITY FLAW - TODO - WE WILL FIX THIS!
-    $result = $pdo->query('SELECT * FROM pet LIMIT '.$limit);
+    $query = 'SELECT * FROM pet';
+    if ($limit != 0) {
+        $query = $query .' LIMIT '.$limit;
+    }
+    $result = $pdo->query($query);
     $pets = $result->fetchAll();
 
     return $pets;
