diff --git a/lib/functions.php b/lib/functions.php
index 9a6ca4b..e058d48 100644
--- a/lib/functions.php
+++ b/lib/functions.php
@@ -11,7 +11,7 @@ function get_pets($limit)
     );
     // THIS IS A HUGE SECURITY FLAW - TODO - WE WILL FIX THIS!
     $query = 'SELECT * FROM pet';
-    if ($limit != 0) {
+    if ($limit) {
         $query = $query .' LIMIT '.$limit;
     }
     $result = $pdo->query($query);
