diff --git a/lib/functions.php b/lib/functions.php
index 7bbe052..558d076 100644
--- a/lib/functions.php
+++ b/lib/functions.php
@@ -11,7 +11,7 @@ function get_pets($limit)
     );
     // THIS IS A HUGE SECURITY FLAW - TODO - WE WILL FIX THIS!
     $sql = 'SELECT * FROM pet';
-    if ($limit != 0) {
+    if ($limit) {
         $sql = $sql .'LIMIT '.$limit;
     }
     $result = $pdo->query($sql);
