diff --git a/lib/functions.php b/lib/functions.php
index 946807b..f8f8335 100644
--- a/lib/functions.php
+++ b/lib/functions.php
@@ -10,7 +10,6 @@ function get_pets()
 
 function save_pets($petsToSave)
 {
-    var_dump($petsToSave);die;
-    $json = json_encode($pets, JSON_PRETTY_PRINT);
+    $json = json_encode($petsToSave, JSON_PRETTY_PRINT);
     file_put_contents('data/pets.json', $json);
 }
diff --git a/pets_new.php b/pets_new.php
index 920ddb3..cf27aae 100644
--- a/pets_new.php
+++ b/pets_new.php
@@ -38,7 +38,7 @@ if ($_SERVER['REQUEST_METHOD'] == 'POST') {
     );
     $pets[] = $newPet;
 
-    save_pets('this is some text!');
+    save_pets($pets);
 
     header('Location: /');
     die;
