diff --git a/pets_new.php b/pets_new.php
index 915bdaa..518c499 100644
--- a/pets_new.php
+++ b/pets_new.php
@@ -27,6 +27,14 @@ if ($_SERVER['REQUEST_METHOD'] == 'POST') {
     }
 
     $pets = get_pets();
+    $newPet = array(
+        'name' => $name,
+        'breed' => $breed,
+        'weight' => $weight,
+        'bio' => $bio,
+        'image' => '',
+        'age' => '',
+    );
     var_dump($name, $breed, $weight, $bio);die;
 }
 ?>
